.changeset/dull-stingrays-mate.md

```diff
@@ -0,0 +1,6 @@
+---
+"@khanacademy/wonder-blocks-i18n": minor
+"@khanacademy/wonder-blocks-form": patch
```

Comment:
**nit(no change required):** You probably don't need to bump `form` (dev-only changes), but I also get if you do it in this case.

Reply:
Good call - removing!